presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -804,6 +815,116 @@ public void testSchemaEvolution()
         assertUpdate("DROP TABLE test_schema_evolution");
     }
 
+    @Test
+    public void testRcAndTextFormats()
+            throws IOException
+    {
+        try {
+            getQueryRunner().execute("CREATE TABLE IF NOT EXISTS lineitem_partition WITH (format='ORC', partitioned_by=ARRAY['ds']) AS " +
```

Comment:
- remove `IF NOT EXISTS`
- move this statement outside of `try` block; otherwise, if CREATE TABLE fails, DROP TABLE in finally block will fail as well